config-model/src/main/java/com/yahoo/searchdefinition/derived/DerivedConfiguration.java

```diff
@@ -50,7 +52,7 @@ public DerivedConfiguration(Search search,
                                 RankProfileRegistry rankProfileRegistry,
                                 QueryProfileRegistry queryProfiles,
                                 ImportedMlModels importedModels) {
-        this(search, new BaseDeployLogger(), rankProfileRegistry, queryProfiles, importedModels);
+        this(search, new BaseDeployLogger(), new TestProperties(), rankProfileRegistry, queryProfiles, importedModels);
```

Comment:
Why is a new TestProperties instance created instead of injecting the actual production implementation?

Reply:
It is only used for testing (now also package private)